### PR TITLE
Shorten well representation in commands list

### PR DESCRIPTION
**Pull request by astaff**
- Adds `stringify_location` to `commands/commands.py` to turn location
  into a string containing an abbreviated list of wells (i.e. 'A1..H12')
  and a position on a deck it is located at. Exact wording and mentioning
  of labware type is subject to user testing.
- Adds trivial tests and updates existing tests where string representation
  of a location is being used.

__NOTE:__ Calls like `container.rows(0, 1, 2)` would produce `WellSeries` of `WellSeries`
this is handled in a suboptimal way (if using lists of lists this could be folding).
This should be re-visited once we phase out `WellSeries`.

# Review request

Please think of any common cases not addressed by this representation.

Here's an excerpt from Bradford Assay with the change in place:
image

### CODE REVIEW

File: api/opentrons/commands/commands.py [hunk 1]
```diff
@@ -2,12 +2,69 @@
...
+            isinstance(location, tuple)):
+        location = [location]
+
+    multiple = len(location) > 1
+
+    return '{object_text}{suffix} {first}{last} in "{slot_text}"'.format(
+            object_text=type_to_text[type(location[0])],
+            suffix='s' if multiple else '',
+            first=location[0].get_name(),
+            last='..'+location[-1].get_name() if multiple else '',
```
Comment (mcous) [inline api/opentrons/commands/commands.py]: Little question: is the selection of `..` vs `...` deliberate here? In some languages (Swift?) `..` means non-inclusive range, which I don't think we're trying to say

Comment (astaff) [reply api/opentrons/commands/commands.py]: I am not sure if our users are aware of that notation. I was using whichever was shorter.

Comment (mcous) [reply api/opentrons/commands/commands.py]: Any problem with using `...`? `..` kinda looks like a typo to me hah

Comment (astaff) [reply api/opentrons/commands/commands.py]: Fixed in e373aa7b2e4457d032e9753906d9747a87a6392a ...

**Pull request merged**